website/docs/docs/cloud/manage-access/invite-users.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Invite users to dbt Cloud"
+description: "Learn how to manually invite users to dbt Cloud"
+id: "invite-users"
+sidebar: "Invite users"
+---
+
+dbt Cloud makes it easy to invite new users to your environment out of the box. This feature is available to all dbt Cloud customers on Teams or Enteprise plans (Developer plans are limited to a single user).
+
+## Invite new users
+
+You must have proper permissions to invite new users:
+
+- [**Teams accounts**](/docs/cloud/manage-access/self-service-permissions) &mdash; must have `member` or `owner` permissions.
+- [**Enterprise accounts**](/docs/cloud/manage-access/enterprise-permissions) &mdash; must have `admin`, `account admin`, `project creator`, or `security admin` permissions.
+
+To invite users: 
+
+1. Open the **Account Settings** from the gear menu in your dbt Cloud account.
+2. From the left sidebar, select **Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/users-screen.png" title="The user screen in settings"/>
+
+3. Click on **Invite Users**.
+
+<Lightbox src="/img/docs/dbt-cloud/access-control/invite-users.png" title="The invite users pane"/>
+
+4. In the **Email Addresses** field, enter the email addresses of the users you would like to invite separated by comma, semicolon, or a new line.
+5. Select the license type for the batch of users from the **License** dropdown.
+6. Select the group(s) you would like the invitees to belong to.
+7. Click **Send Invitations**.
+    - If the list of invitees exceeds the number of licenses your account has availabe, you will receive a warning when you click **Send Invitations** and the invitations will not be sent. 
+
+
+## User experience
+
+dbt generates and sends the email to the addresses defined in dbt Cloud. Be sure to allow traffic from `support@getdbt.com` to prevent emails from going to spam or being blocked (this is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/regions-ip-addresses)).
+
+The email contains a link to create an account. When the user clicks on this they will be brought to one of two screens depending on whether SSO is configured or not.
```

Comment:
maybe the two diff flows can present in tabs so it's clearer and users can go directly to their use case?